SilMock/DataStore/Sqlite/SqliteUtils.php

```diff
@@ -153,42 +157,39 @@ public function deleteRecordById($recordId)
      * A utility function to delete the Google Mock data based on a
      * particular data type and data class for a specific email.
      *
-     * @param string $dataType, the name of a Google mock service (e.g. 'directory')
-     * @param string $dataClass, the name of a Google mock class (e.g. 'users_alias')
+     * @param string $dataType -- the name of a Google mock service (e.g. 'directory')
+     * @param string $dataClass -- the name of a Google mock class (e.g. 'users_alias')
```

Comment:
```suggestion
     * @param string $dataType the name of a Google mock service (e.g. 'directory')
     * @param string $dataClass the name of a Google mock class (e.g. 'users_alias')
```

Reply:
I prefer it with. The visual marker is useful for making the semantic break apparent.